### ISSUE TITLE
Obey user settings for showing suggestions

### ISSUE THREAD

**Issue by ghost** [aioutecism/amVim-for-VSCode#41]
There's an "editor.quickSuggestions" setting.  If the user has disabled it, we shouldn't indiscriminately show the suggestions box like we do right now.

````
The following changes since commit dc564bd6180744fe9cf203994e1aae124010c58c:

  Update version to 0.1.1. (2016-02-10 07:22:51 +0900)

are available in the git repository at:

  https://gitlab.com/crussell/amVim.git suggestion-setting

for you to fetch changes up to 2e6495df3c893ca709ad1c89716be090353cab86:

  Respect "editor.quickSuggestions" user setting (2016-02-12 00:33:52 -0600)

----------------------------------------------------------------
Colby Russell (1):
      Respect "editor.quickSuggestions" user setting

 src/Actions/Suggestion.ts | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)
````

https://gitlab.com/crussell/amVim/commit/2e6495df3c893ca709ad1c89716be090353cab86

**Comment by aioutecism** [comment]
Commented https://gitlab.com/crussell/amVim/commit/2e6495df3c893ca709ad1c89716be090353cab86#note_3697706

**Comment by colbyrussell** [comment]
Addressed review comments: https://gitlab.com/crussell/amVim/commit/39a6058ece5b3f57f3bf3a3aebe2ac654bf01706

**Comment by aioutecism** [comment]
Merged 5122de744bd9788e8bb71b63acbec92e16b3c595
